### ISSUE TITLE
Longitudinal erythronychia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10633]
**Preferred term label:**

**Synonyms**
longitudinal red stripe

**Definition (free text, please give PubMed ID)**
PMID: 38705197 Longitudinal erythronychia (LE) is defined as a longitudinal red band extending from the proximal nail fold or lunula to the distal nail plate. The differential diagnosis is limited and distinct for localized longitudinal erythronychia (LLE) (involving 1 nail) and polydactylous longitudinal erythronychia (PLE) (involving>1 nail).

PMID: 30137841 Darier disease, previously known as keratosis follicularis, is a genodermatosis inherited in an autosomal dominant fashion. This rare disease classically features multiple red or brown papules with hyperkeratosis, nail abnormalities (eg, **longitudinal erythronychia**), and mucosal changes that present initially around puberty and are exacerbated throughout the lifespan, usually with external exposures, such as sunlight.

**Parent term (use hpo.jax.org/app)**
Abnormality of the nail HP:0001597

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:218 DARIER DISEASE
ORPHA:391651 Glomus tumor

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001074
- label: Longitudinal erythronychia
- parents: [Abnormality of the nail (HP:0001597)](https://hpo.jax.org/app/browse/term/HP:0001597)
- pmid: PMID:38705197|PMID:30137841
- def: A longitudinal red band extending from the proximal nail fold or lunula to the distal nail plate.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10633
- synonyms:
